src/CoffeeShop/CoffeeShop.php

```diff
@@ -0,0 +1,30 @@
+<?php
+namespace CoffeeShop;
+use base\Location;
+
+class CoffeeShop extends Location implements IShop{
+    private $__shopName;
+
+    public function __construct($xCoord, $yCoord, $shopName)
+    {
```

Comment:
The style is inconsistent, sometimes you use `{` on a new line sometime you put then on the same line.

Reply:
I don't like to blame others for my mistakes.
My style is with ` { ` on the same line.
Sometime is happening that my editor wants to help me write faster. I should train it well.